lib/variables.less

```diff
@@ -247,6 +247,74 @@
   margin-top: -1px;
 }
 
+.loader(@color: #ffffff, @spinner: @text-base) {
+  font-size: 10px;
+  margin: 50px auto;
+  text-indent: -9999em;
+  width: 7em;
+  height: 7em;
+  border-radius: 50%;
+  background: @spinner;
+  background: -moz-linear-gradient(left, @spinner 10%, rgba(255, 255, 255, 0) 42%);
+  background: -webkit-linear-gradient(left, @spinner 10%, rgba(255, 255, 255, 0) 42%);
+  background: -o-linear-gradient(left, @spinner 10%, rgba(255, 255, 255, 0) 42%);
+  background: -ms-linear-gradient(left, @spinner 10%, rgba(255, 255, 255, 0) 42%);
+  background: linear-gradient(to right, @spinner 10%, rgba(255, 255, 255, 0) 42%);
+  position: relative;
+  -webkit-animation: load 1.4s infinite linear;
+  animation: load 1.4s infinite linear;
+  -webkit-transform: translateZ(0);
+  -ms-transform: translateZ(0);
+  transform: translateZ(0);
+
+  &:before {
+    width: 50%;
+    height: 50%;
+    background: @spinner;
+    border-radius: 100% 0 0 0;
+    position: absolute;
+    top: 0;
+    left: 0;
+    content: '';
+  }
+
+  &:after {
+    background: @color;
+    width: 75%;
+    height: 75%;
+    border-radius: 50%;
+    content: '';
+    margin: auto;
+    position: absolute;
+    top: 0;
+    left: 0;
+    bottom: 0;
+    right: 0;
+  }
+}
+
+@-webkit-keyframes load {
+  0% {
+    -webkit-transform: rotate(0deg);
+    transform: rotate(0deg);
+  }
+  100% {
+    -webkit-transform: rotate(360deg);
+    transform: rotate(360deg);
+  }
+}
+
+@keyframes load {
+  0% {
+    -webkit-transform: rotate(0deg);
```

Comment:
`-ms` prefix missing here...use the existing `.transform()` mixins (in _variables.less_)